docs/release.rst

```diff
@@ -14,12 +14,25 @@ Unreleased
     # .. warning::
     #    Pre-release! Use :command:`pip install --pre zarr` to evaluate this release.
 
+
+Major changes
+~~~~~~~~~~~~~
+
 * Improve Zarr V3 support, adding partial store read/write and storage transformers.
-  Add two features of the [v3 spec](https://zarr-specs.readthedocs.io/en/latest/core/v3.0.html):
-  * storage transformers
-  * `get_partial_values` and `set_partial_values`
+  `v3 spec <https://zarr-specs.readthedocs.io/en/latest/core/v3.0.html>`_.
   By :user:`Jonathan Striebel <jstriebel>`; :issue:`1096`.
 
+  - storage transformers
+
+  - ``get_partial_values`` and ``set_partial_values``
```

Comment:
So this looks ok in reStructuredText, but Sphinx seems to render it wacky. In any event at least this is consistent with what [we do elsewhere]( https://github.com/zarr-developers/zarr-python/blob/b9e9f5aaa44ca564835c3c242937236ea04aecb2/docs/release.rst#maintenance-4 )